src/EnsembleKalmanProcess.jl

```diff
@@ -193,23 +206,36 @@ function EnsembleKalmanProcess(
     # timestep store
     Δt = FT[]
 
+    # set up accelerator
+    if isnothing(accelerator)
+        acc = DefaultAccelerator()
+    else
+        acc = accelerator
+    end
+    AC = typeof(acc)
+
+    if AC <: NesterovAccelerator
+        set_ICs!(acc, params)
+    end
+
     # failure handler
     fh = FailureHandler(process, failure_handler_method)
     # localizer
     loc = Localizer(localization_method, N_par, N_obs, N_ens, FT)
 
     if verbose
-        @info "Initializing ensemble Kalman process of type $(nameof(typeof(process)))\nNumber of ensemble members: $(N_ens)\nLocalization: $(nameof(typeof(localization_method)))\nFailure handler: $(nameof(typeof(failure_handler_method)))\nScheduler: $(nameof(typeof(lrs)))"
+        @info "Initializing ensemble Kalman process of type $(nameof(typeof(process)))\nNumber of ensemble members: $(N_ens)\nLocalization: $(nameof(typeof(localization_method)))\nFailure handler: $(nameof(typeof(failure_handler_method)))\nScheduler: $(nameof(typeof(lrs)))\nAccelerator: $(nameof(typeof(acc)))"
```

Comment:
I would ignore testing this as it is a print statement (just run it once locally with the flag verbose=true to make sure it runs